src/main/java/org/fundacionjala/coding/nestor/Price.java

```diff
@@ -0,0 +1,6 @@
+package org.fundacionjala.coding.nestor;
+
+public abstract class Price {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "Price" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)